partitioner_test.go

```diff
@@ -120,3 +121,57 @@ func TestManualPartitioner(t *testing.T) {
 		}
 	}
 }
+
+// By default, Sarama uses the message's key to consistently assign a partition to
+// a message using hashing. If no key is not, a random partition will be chosen.
+// This example shows how you can partition messages randomly, even when a key is set
+func ExamplePartitioner_random() {
```

Comment:
not sure how useful this example is, it's literally just the one line config change...